implementations/micrometer-registry-datadog/build.gradle

```diff
@@ -1,9 +1,8 @@
-apply plugin: 'nebula.optional-base'
-
 dependencies {
-    compile project(':micrometer-core')
-    compile 'org.slf4j:slf4j-api:1.7.+'
-    compile 'org.apache.commons:commons-text:latest.release'
+    api project(':micrometer-core')
+
+    api 'org.apache.commons:commons-text:latest.release'
```

Comment:
I'm not seeing this used at all from a quick search, but I can try removing that separate of this pull request. Reviewing the changes, it just caught my eye.